devel/symfony-cli/Portfile

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+version             5.5.2
+revision            0
+
+if {${os.major} >= 17} {
+    set source_build    yes
+} else {
+    set source_build    no
+}
+
+if ${source_build} {
+    PortGroup           golang 1.0
+
+    go.setup            github.com/symfony-cli/symfony-cli ${version} v
+} else {
+    PortGroup           github 1.0
+
+    github.setup        symfony-cli symfony-cli ${version} v
+}
+
+name                symfony-cli
+
+homepage            https://symfony.com/download
+
+description         The Symfony CLI tool
+
+long_description    The Symfony CLI tool is a must-have tool when developing \
+                    Symfony applications on your local machine.
+
+platforms           darwin
```

Comment:
`platforms darwin` is now implicit in macOS. You can actually remove this line.

Reply:
removed